exchange/exchange.go

```diff
@@ -1102,6 +1089,33 @@ func applyCategoryMapping(ctx context.Context, targeting openrtb_ext.ExtRequestT
 	return res, seatBids, rejections, nil
 }
 
+// findDurationRange returns the element in the array 'a' that is both greater than 'dur' and closest
+// in value to 'dur' unless a value equal to 'dur' is found. Returns an error if all elements in 'a'
+// are less than 'dur'.
+func findDurationRange(dur int, a []int) (int, error) {
```

Comment:
Could the array variable `a` have a more descriptive name? Or are you keeping it more general in case this function is re-used in another place? If that's the case, could this function be moved to `utils.go`?